test/EndToEnd/tests/TabExpansionTest.ps1

```diff
@@ -282,7 +282,6 @@ function Test-TabExpansionForVersionForUninstallPackage {
     # Arrange
     $p = New-WebApplication
     $p | Install-Package elmah -Version 1.1
```

Comment:
Could we use this opportunity to move all our tests to .NET or Azure SDK packages?

Reply:
Just looked at "elmah" package and its being used 156 times in all kind of files, even in [commmand line outputs ](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.Designer.cs#L715)😨 

Maybe that task is big and we will need to create another task for that, I suggest fixing this asap first

Reply:
Could you please create an issue to track that?

Reply:
@aortiz-msft Created the issue in https://github.com/NuGet/Client.Engineering/issues/2434.